### ISSUE TITLE
Re:setting local env

### ISSUE THREAD

**Issue by idpokute** [WordPress/gutenberg#4055]
## Issue Overview

I followed the local environment section in the contributing guide; I installed docker and ./bin/setup-local-env.sh. After some configuration for WSL, I installed Wordpress, and run npm build command. But I can't find Gutenberg under Plugin. There are only HelloDolly and Akismet. I'm not sure whether this is the docker issue or normal. Can anyone give some advice? Thank you.

**Comment by idpokute** [comment]
Does anyone accomplish setting-up local environment with Docker on WSL before?

**Comment by youknowriad** [comment]
@idpokute Yes, I use this local environment.

>  After some configuration for WSL, I installed Wordpress,

Curious, what configuration are you talking about. You don't need to do anything aside `npm install && npm run build` locally. Everything (installing WP and activating the plugin) is already done in `./bin/setup-local-env.sh`

Once you run this command, you can just go to `http://localhost:8888` and it should work.

**Comment by idpokute** [comment]
@youknowriad 
When I test it, I changed port to 2357 for daemon. 
Now I set the default value, and test it on my desktop at work. Still no luck.

Here is the error message on Ubuntu on Windows 10 (WSL)
./bin/setup-local-env.sh
ERROR: readlink /var/lib/docker/overlay2/l: invalid argument

Here is the same error message on Gitbash on Windows 10
$ ./bin/setup-local-env.sh
readlink /var/lib/docker/overlay2/l: invalid argument

Did you use WSL? or pull and run it on Windows10 without WSL?
